core/src/main/java/com/novoda/support/MerlinBackwardsCompatibleLog.java

```diff
@@ -0,0 +1,60 @@
+package com.novoda.support;
+
+public final class MerlinBackwardsCompatibleLog implements Logger.LogHandle {
+
+    private static MerlinBackwardsCompatibleLog lazyInstance;
+
+    private final String TAG = "Merlin" + hashCode();
+
+    private MerlinBackwardsCompatibleLog() {
+        // Single instance.
+    }
+
+    public static MerlinBackwardsCompatibleLog getInstance() {
```

Comment:
Lazy single instance. One instance so only one is guaranteed to be present in the `Logger` at any time.